include/util/byteswap.h

```diff
@@ -28,18 +28,30 @@
 
 CPP_GUARD_BEGIN
 
-//! Byte swap unsigned short
+/*
+ * Byte swap unsigned 16 bits
+ */
 uint16_t swap_uint16( uint16_t val );
 
-//! Byte swap short
+/*
+ * Byte swap signed 16 bits
+ */
 int16_t swap_int16( int16_t val );
```

Comment:
we actually had it before, I re-organized the code and updated the documentation, which made it look like new code. :)